website/source/layouts/layout.erb

```diff
@@ -76,7 +76,7 @@
                     </ul>
                   </li>
                   <li><a href="/intro/index.html">Intro</a></li>
-                  <li><a href="/docs/guides/index.html">Guides</a></li>
+                  <li><a href="https://learn.hashicorp.com/consul?utm_source=consul.io&utm_content=top-nav">Learn</a></li>
```

Comment:
utm+medium=docs

Reply:
My thought was, we can still see all traffic from `utm_source=consul.io` including the docs links and top nav change. But if we change the second utm parameter to `utm_content=top-nav` then we can differentiate even further where traffic is coming from. I was thinking a little ahead on this since the main goal is consul.io generally. Also, I think it's still neat to have extra granularity. This is also why I changed the side nav to `utm_content=guide-index`, then we can measure if more people are using the top nav or guide side nav. I think ultimately we'd want to just remove the "guide" side nav or have it direct users to Learn.

Reply:
fully agree on the extra granularity, i just think we should have all three

source = consul.io
medium = docs
content = top-nav

so when we do the JTBD experiement it would be 

source = consul.io
medium = home-page
content = jtbd